src/content/docs/guides/Web development.mdx

```diff
@@ -148,12 +150,81 @@ Tutaj mamy dwie ścieżki:
 
 1. React Server Components
 
-   Domyślna ścieżka, nie trzeba zarządzać stanami ładowania, wszystko jest od razu na miejscu, bardzo wygodne
+   React Server Components (RSC) to domyślne podejście w Next.js 13+. Dzięki nim dane są pobierane po stronie serwera, co eliminuje konieczność zarządzania stanami ładowania i zapewnia szybsze ładowanie stron. Zaletami jest automatyczne renderowanie danych na serwerze, brak konieczności zarządzania stanami ładowania i są proste w użyciu.
+
+    ```jsx
+    // app/products/page.tsx
+    import { db } from '@/lib/db';
```

Comment:
hmm, zastanawiam się na ile te przykłady z db są dobre, raczej w solvro wszędzie fetchujemy z api w tym miejscu, prosty fetch chyba byłby lepszy, ten syntax to bardziej prismowy i nigdzie tego tu nie ma 😭